source/agora/consensus/protocol/Nominator.d

```diff
@@ -556,6 +563,61 @@ extern(D):
         this.active_timers[type] = this.taskman.setTimer(
             timeout.msecs, { callCPPDelegate(callback); });
     }
+
+    /***************************************************************************
+
+        Used by the nomination protocol to randomize the order of messages
+        between nodes.
+
+        Params:
+            slot_idx = the slot index we're currently reaching consensus for.
+            prev = the previous data set for the provided slot index.
+            is_priority = the flag to check that this call is for priority.
+            round_number = the nomination round
+            node_id = the id of the node for which this computation is being made
+
+    ***************************************************************************/
+
+    public override uint64_t computeHashNode(uint64_t slot_idx,
```

Comment:
No document definition for the return value. `uint64_t`

Reply:
No document definition for the return value. uint64_t

> I will add a description for that. Thanks.